src/Angor/Client/Pages/Create.razor

```diff
@@ -204,293 +189,425 @@
-                </div>
-                <div class="d-flex align-items-center">
-                    <Icon IconName="calendar"></Icon>
-                    <p class="card-text ms-2 mb-1"><strong>Stage Date:</strong> @stage.ReleaseDate.ToString("dd/MM/yyyy") - @((stage.ReleaseDate - project.ProjectInfo.StartDate).Days) days after project starts</p>
-                </div>
-            </div>
+            <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+            <span>Creating Nostr Profile...</span>
+        }
+        else
+        {
+            <span>Create Nostr Profile</span>
         }
+    </button>
 
-        <div class="card card-body mt-4">
-            <div class="d-flex justify-content-between">
-                <button type="button" class="btn btn-success" @onclick="() => activeTab = 2">
-                    <span>Previous</span>
-                </button>
-                <button type="button" class="btn btn-success" @onclick="CreatProjectTransactionAnCheckPassword" disabled="@creatProjectSpinner">
-                    @if (creatProjectSpinner)
-                    {
-                        <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
-                        <span>Next...</span>
-                    }
-                    else
-                    {
-                        <span>Next</span>
-                    }
-                </button>
-            </div>
+    <button class="btn btn-primary mt-4" @onclick="ConfirmFinalizeProjectInfoCreation" disabled="@(!nostrMetadataCreated || nostrApplicationSpecificDataCreated)">
+        @if (createApplicationDataSpinner)
+        {
+            <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+            <span>Creating Project Info...</span>
+        }
+        else
+        {
+            <span>Create Project Info</span>
+        }
+    </button>
+    <div class="card card-body mt-4">
+        <div class="d-flex justify-content-between">
+            <button type="button" class="btn btn-success" @onclick="() => ChangeTab(2)">
+                <span>Previous</span>
+            </button>
+            <button type="button" class="btn btn-success" @onclick="CreatProjectTransactionAnCheckPassword" disabled="@(creatProjectSpinner || !nostrApplicationSpecificDataCreated || !nostrMetadataCreated)">
+                @if (creatProjectSpinner)
+                {
+                    <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+                    <span>Create Project...</span>
+                }
+                else
+                {
+                    <span>Create Project</span>
+                }
+            </button>
+
+            <button class="btn btn-danger ms-3"
+                    @onclick="@(() => ShowResetConfirmation("nostrMetadataCreated"))"
```

Comment:
Reset info doesn't seem to work, I think it should move you back to the info page
Also it is not clear what button resets the metadata, do we need the reset buttons? a user can just go back and do it manually on the tab they need, it might be too confusing IMO.

Reply:
once you create a nostr metada, or project info (which is UI only), you are restricted from changing them.

mainly because of the nostr one that once its created it's on nostr protocol.

i added these buttons as a "fail safe", incase there's an error that isn't from nostr, or because of something unexpected. should we do it in another way?

Reply:
yes I think we don't need to have this reset, once the creator posts the metadata to nostr then we must disable every thing else and only enable the button to publish to the bitcoin network.

A user can always just navigate backwards and make the changes they want anyway.
The reset just makes it confusing.